src/phoenix/experimental/evals/evaluators.py

```diff
@@ -0,0 +1,172 @@
+import json
+from dataclasses import dataclass
+from typing import Any, List, Mapping, Optional, Protocol, cast
+
+from .models.openai import OpenAIModel, OpenAIResponse
+from .templates import PromptTemplate
+
+Record = Mapping[str, Any]
```

Comment:
Not clear what the key or the output of this record is. Can you add a bit more color or type aliasing to make sure it's clear what this is doing.